src/src.pro

```diff
@@ -8,7 +8,7 @@ PLUGIN_IMPORT_PATH = io/thp/pyotherside
 
 TEMPLATE = lib
 CONFIG += qt plugin
-QT += qml quick
+QT += qml quick svg
```

Comment:
Ok, let's leave that in here then and we can always work around it / fix it / conditionalize it later when there's a need for it.